src/endless/res/EndlessUsbTool.htm

```diff
@@ -307,9 +312,12 @@
                 
 					<div id="SecureBootReminder" class="RoundedCornersContainer">
 						<div id="ReminderTextContainer">
-                            <div class="ReminderTitle" id="ThankYouDualBootReminder">Restart your computer and select "Endless OS" from the menu.</div>
-                            <div class="ReminderTitle" id="ThankYouLiveReminder">To try Endless OS now, restart your computer.</div>
-                            <div class="ReminderTitle" id="ThankYouReflasherReminder">Now you're ready to install Endless OS on more computers.</div>
+                            <div class="ReminderTitle ModeNormal" id="ThankYouDualBootReminder">Restart your computer and select "Endless OS" from the menu.</div>
+                            <div class="ReminderTitle ModeCoding" id="ThankYouDualBootReminderCoding">Restart your computer and select "Endless Code" from the menu.</div>
```

Comment:
FWIW, I have found one occurrence in [eos-image-builder:data/grub.cfg](https://github.com/endlessm/eos-image-builder/blob/master/data/grub.cfg#L108-L110), but it is a menu entry just showing a warning message about Endless OS not being available while Windows is hibernated. :rofl: